In-house patch created to clear any merge conflicts from source in archive
repo to build properly on Solaris.

--- CACKey-0.7.4/autogen.sh	2016-05-19 16:20:45.459266138 -0400
+++ CACKey-0.7.4/autogen.sh	2015-07-16 12:20:20.000000000 -0400
@@ -6,8 +6,6 @@
 autoconf; autoheader
 
 rm -rf autom4te.cache/
-<<<<<<< HEAD
-=======
 
 for basefile in install-sh config.sub config.guess; do
 	for path in /usr/share/automake-*; do
@@ -20,4 +18,3 @@
 		fi
 	done
 done
->>>>>>> trunk
